drake/solvers/Constraint.m

```diff
@@ -78,6 +80,18 @@
       obj.nnz = numel(iCfun);
     end
 
+    function [iCfun,jCvar,nnz] = getSparseStructure(obj)
```

Comment:
Maybe rename this to getGradientSparsityPattern? I see that there was already a matching setSparseStructure, so I understand why you chose this, but still.